lib/main.js

```diff
@@ -341,19 +449,45 @@ exports.main = function () {
     contentScript: 'self.on("click", self.postMessage)',
     onMessage: launchConfigTab
   });
+
   checkUpdate();
   setUpTopMenu(initSubMenus());
+
+  // Adding a simple playlist.
+  plists.playlist_id = plists.add_playlist();
+  plists.playlist_id = plists.add_playlist();
```

Comment:
Hum, I have to check, but I think it was because (I suppose, thats may be false) I have index problems and 0 didn't work. So I add it twice in order to begin with 1. I will check again and fix this if I don't have anymore issues.